### PR TITLE
Change `GCEDiskAlphaAPI` to `DiskAlphaAPI` in comments

### PR DESCRIPTION
**Pull request by saad-ali**
Fixes https://github.com/kubernetes/kubernetes/issues/52258

### CODE REVIEW

File: pkg/cloudprovider/providers/gce/gce.go [hunk 1]
```diff
@@ -830,7 +830,7 @@ func (manager *GCEServiceManager) CreateRegionalDisk(
 			manager.gce.projectID, manager.gce.region, diskToCreateAlpha).Do()
 	}
 
-	return nil, fmt.Errorf("The regional PD feature is only available via the GCE Alpha API. Enable \"GCEDiskAlphaAPI\" in the list of \"alpha-features\" in \"gce.conf\" to use the feature.")
+	return nil, fmt.Errorf("The regional PD feature is only available via the GCE Alpha API. Enable %q in the list of \"alpha-features\" in \"gce.conf\" to use the feature.", GCEDiskAlphaFeatureGate)
```
Comment (msau42) [inline pkg/cloudprovider/providers/gce/gce.go]: Do you want to rename GCEDiskAlphaFeatureGate to be consistent with the networking feature that's also in the enum?

**Comment by spiffxp** [comment]
@saad-ali @msau42 it seems like presubmits are hanging at the time I comment, is there anything left to be done on this PR otherwise?

**Comment by dims** [comment]
@kubernetes/sig-storage-pr-reviews PTAL

File: pkg/cloudprovider/providers/gce/gce.go (hunk 1 above)
Comment (saad-ali) [reply pkg/cloudprovider/providers/gce/gce.go]: Renamed to `AlphaFeatureGCEDisk` to align with the networking feature enum

**Comment by saad-ali** [comment]
@msau42 PTAL

**Comment by msau42** [comment]
I believe this is still needed

File: pkg/cloudprovider/providers/gce/gce.go (hunk 1 above)
Comment (saad-ali) [reply pkg/cloudprovider/providers/gce/gce.go]: This was also done in https://github.com/kubernetes/kubernetes/pull/51067, rebasing.

**Pull request merged**